src/yum-plugins/tracer_upload.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import json
+import httplib
+import urllib
+import time
+
+from yum.plugins import PluginYumExit, TYPE_CORE, TYPE_INTERACTIVE
+from rhsm.config import RhsmConfigParser, initConfig
+from tracer import Query
```

Comment:
If you can catch the import error and throw some sort of warning, that would probably be ideal.  We'd want to exclude it from <el7 on the spec file regardless i think.